MapView/Map/RMBackgroundOperationWorker.m

```diff
@@ -0,0 +1,77 @@
+//
+//  RMBackgroundOperationWorker.m
+//  MapView
+//
+//  Created by Natalia Paula Osiecka on 22/09/16.
+//  Copyright (c) 2014 Ordnance Survey. All rights reserved.
+//
+
+#import "RMBackgroundOperationWorker.h"
+
+@interface RMBackgroundOperationWorker()
+
+@property(nonatomic, readonly) NSArray<id> *observerIds;
```

Comment:
Is there any value to the generic annotation here? I would think `NSArray *` is equivalent to `NSArray<id> *`, so perhaps being concise is preferable?